libafl_qemu/src/modules/calls.rs

```diff
@@ -412,8 +412,12 @@ where
         );
     }
 
-    fn pre_exec<ET>(&mut self, emulator_modules: &mut EmulatorModules<ET, S>, input: &S::Input)
-    where
+    fn pre_exec<ET>(
+        &mut self,
+        emulator_modules: &mut EmulatorModules<ET, S>,
+        input: &S::Input,
+        _state: &mut S,
```

Comment:
State should be first again, see also:

https://github.com/AFLplusplus/LibAFL/blob/4c5df53c6dac55525d3f68314f62d4d6f6ce842d/libafl/src/observers/mod.rs#L59